### ISSUE TITLE
File Upload: When using Form Item with File Upload the label isn't read by screen readers

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#2370]
# Info

This was discovered by Dustin during random accessibility testing

All you need to replicate is a Form Item component surrounding a File Upload component.

If you use a screen reader in this scenario. The screen reader properly reads off that it is a clickable button marked as "Choose Files", and that no files have been chosen. But it doesn't read the Form Item label associated with that Input.

## Acceptance Criteria:

1. A screen reader reads the Form Item label surrounding a File Upload component

**Comment by ArakTaiRoth** [comment]
Just confirming that this is still an issue
